fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/processor/impl/CosExportImpl.java

```diff
@@ -161,13 +163,48 @@ public Parameters exportPatient(String logicalId, MediaType outputFormat, Instan
         }
     }
 
-    // not implemented yet
     @Override
     public Parameters exportGroup(String logicalId, MediaType outputFormat, Instant since,
         List<String> types, List<String> typeFilters, FHIRRequestContext ctx,
         FHIRResourceHelpers resourceHelper, FHIROperationContext operationContext,
         BulkDataTenantSpecificCache cache) throws FHIROperationException {
-        throw new FHIROperationException("No $export group operation right now");
+
+        try {
+            log.fine("Using the COS Implementation");
+
+            Map<String, String> properties =
```

Comment:
just a note to Albert... I'll change this in my PR.  so we don't just force DEFAULT_TENANT_ID

Reply:
sounds great!! thanks!